bin/gwdetchar-lasso-correlation

```diff
@@ -199,6 +201,11 @@ else:
 if args.remove_outliers:
     logger.debug("-- Removing outliers above %f sigma" % args.remove_outliers)
     gwlasso.remove_outliers(primaryts, args.remove_outliers)
+elif args.remove_outliers_pf:
+    logger.debug("-- Removing outliers in the bottom %f percent of data" % args.remove_outliers_pf)
```

Comment:
The `flake8` linter is complaining that this line is too long, can you break it up across two lines and use string formatting like so?
```suggestion
    logger.debug("-- Removing outliers in the bottom {} percent "
                 "of data".format(args.remove_outliers_pf))
```